src/table/components/virtualized-table/ScrollableContainer.tsx

```diff
@@ -0,0 +1,37 @@
+import { stardust } from '@nebula.js/stardust';
+import React from 'react';
+
+interface ScrollableContainerProps {
+  children: (JSX.Element | null)[] | JSX.Element;
+  forwardRef: React.RefObject<HTMLDivElement>;
+  height: number;
+  width: number;
+  constraints: stardust.Constraints;
+  onScroll: (event: React.SyntheticEvent<Element, Event>) => void;
+}
+
+const ScrollableContainer = ({
+  children,
+  width,
+  height,
+  forwardRef,
+  constraints,
+  onScroll,
+}: ScrollableContainerProps): JSX.Element => {
```

Comment:
Shouldn't we use `React.forwardRef(...)` syntax here?

Reply:
Agree, will fix